packages/kbn-grid-layout/grid/grid_panel/drag_handle.tsx

```diff
@@ -50,25 +56,46 @@ export const DragHandle = React.forwardRef<
     },
     [interactionStart, gridLayoutStateManager.accessMode$]
   );
+  const onTouchStart = useCallback(
+    (e: UserTouchEvent) => {
+      e.stopPropagation();
+      interactionStart('drag', e);
+    },
+    [interactionStart]
+  );
+  const onTouchEnd = useCallback(
+    (e: UserTouchEvent) => {
+      e.stopPropagation();
+      interactionStart('drop', e);
+    },
+    [interactionStart]
+  );
 
   const setDragHandles = useCallback(
     (dragHandles: Array<HTMLElement | null>) => {
+      if (gridLayoutStateManager.accessMode$.getValue() !== 'EDIT') {
```

Comment:
I might be missing something here and maybe there's a reason to put it directly in the `onTouchStart` etc callbacks?

Reply:
So the reason I had it in the `onMouseDown` is that the drag handles don't always get unmounted on view mode change - specifically, when it comes to embeddables, this occurs when the panel **title** is acting as the drag handle. So if we don't cancel out drag events **on mouse down and/or touch start**, the user can drag panels around in view mode:


https://github.com/user-attachments/assets/b3ddcc3a-7c7b-4f3a-952c-bbc1fcfdd451